### PR TITLE
[copy update] WD-4532 update heading text on `/data/mongodb`

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Changed the heading text on `/data/mongodb`

## QA

- Check out the [demo here](https://ubuntu-com-12952.demos.haus/data/mongodb) and compare it with the text specified in the [task](https://warthogs.atlassian.net/browse/WD-4532) and the [copy doc](https://docs.google.com/document/d/1_JuxbWsqTeQRRmSL0sCpM3alabr06WX4Cb-tt1BBW_I/edit)
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

- [WD-4532](https://warthogs.atlassian.net/browse/WD-4532)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-4532]: https://warthogs.atlassian.net/browse/WD-4532?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedoch** [comment]
@anthonydillon please take a look if you can. Thank you!

**Comment by pedoch** [comment]
@mtruj013 please a take a look when you can

**Review by mtruj013 — APPROVED** [review]
LGTM, just one suggestion:

File: templates/data/mongodb/index.html [hunk 1]
```diff
@@ -36,12 +36,12 @@ <h1 class="p-heading--2 p-heading--bold">
 <section class="p-strip u-no-margin--bottom u-no-padding--top">
   <div class="row">
     <hr class="u-no-margin--bottom" />
-    <div class="col-medium-3">
+    <div class="col-6 col-medium-3">
```
[the remark this answers is not in the document]
Comment (pedoch) [reply templates/data/mongodb/index.html:39]: Hi @mtruj013 I'm not sure what I'm supposed to fix here. Could you elaborate?

[the remark this answers is not in the document]
Comment (pedoch) [reply templates/data/mongodb/index.html:39]: Okay, got it!

**Pull request merged**